react/SlideToggle/SlideToggle.less

```diff
@@ -0,0 +1,82 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-flex;
+  line-height: @grid-row-height * 3;
+  height: @grid-row-height * 3;
+  font-size: 14px;
+}
+
+.switch {
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+}
+
+.input {
+  position: absolute;
+  left: 10px;
+  top: 5px;
+  margin: 0;
+
+  &:focus + .slider {
+    box-shadow: 0px 0px 10px 0px @sk-focus;
+  }
+
+  &:active + .slider .slideButton {
+    transform: scale(1.15, 0.85);
```

Comment:
Yeah I'll give that a try.  I never intended to animate the icon - you just get the scaling on child elements